### PR TITLE
New feature added:

### PR DESCRIPTION
**Pull request by lzf834**
- find students by their Activities `findactivity` Command

Collaborator: Yong Rui

### CODE REVIEW

**Review by Fenway17 — APPROVED** [review]
Everything seems in order.

File: src/main/java/seedu/address/model/activity/ActivityContainsKeywordsPredicate.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+public class ActivityContainsKeywordsPredicate implements Predicate<Person> {
+    private final List<String> keywords;
+
+    public ActivityContainsKeywordsPredicate(List<String> keywords) {
+        this.keywords = keywords;
+    }
+
+    @Override
+    public boolean test(Person person) {
+        String activitiesString = person.getActivities().toString().replace("[","").replace("]", "").replace(",", "");
```
Comment (Fenway17) [inline src/main/java/seedu/address/model/activity/ActivityContainsKeywordsPredicate.java]: Maybe we can find a better way to implement this, but this works for now.

Comment (lzf834) [reply src/main/java/seedu/address/model/activity/ActivityContainsKeywordsPredicate.java]: Roger, open to future fixes if anyone has a better suggestion for this method here.

File: src/main/java/seedu/address/model/activity/ActivityContainsKeywordsPredicate.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+public class ActivityContainsKeywordsPredicate implements Predicate<Person> {
+    private final List<String> keywords;
+
+    public ActivityContainsKeywordsPredicate(List<String> keywords) {
+        this.keywords = keywords;
+    }
+
+    @Override
+    public boolean test(Person person) {
+        String activitiesString = person.getActivities().toString().replace("[", "")
```
Comment (Fenway17) [inline src/main/java/seedu/address/model/activity/ActivityContainsKeywordsPredicate.java:21]: Maybe we can find a better way to implement this, but works for now.

**Review by Fenway17 — APPROVED** [review]
Everything seems in order.

**Pull request merged**